lib/net/http/requests.rb

```diff
@@ -1,67 +1,318 @@
 # frozen_string_literal: false
-#
+
 # HTTP/1.1 methods --- RFC2616
-#
 
-# See Net::HTTPGenericRequest for attributes and methods.
-# See Net::HTTP for usage examples.
+# \Class for representing
+# {HTTP method GET}[https://en.wikipedia.org/w/index.php?title=Hypertext_Transfer_Protocol#GET_method]:
+#
+#   require 'net/http'
+#   uri = URI('https://jsonplaceholder.typicode.com')
+#   hostname = uri.hostname # => "jsonplaceholder.typicode.com"
+#
+#   req = Net::HTTP::Get.new(uri) # => #<Net::HTTP::Get GET>
+#   # Default headers.
+#   req.to_hash
```

Comment:
These default headers changes depending on whether the user's machine supports gzip compression or not, so I'm not sure how useful it is to show the default value of these headers.

Reply:
Removed (15 places).